tests/e2e/modules/compliments_spec.js

```diff
@@ -123,3 +126,39 @@ describe("Compliments module", function () {
 		});
 	});
 });
+
+describe("Feature Advice", function () {
+
+	let app;
+
+	helpers.setupTimeout(this);
+
+	async function setup(responses) {
+		app = await helpers.startApplication({
+			args: ["js/electron.js"]
+		});
+
+		wdajaxstub.init(app.client, responses);
+
+		app.client.setupStub();
+	}
+
+	afterEach(function () {
+		return helpers.stopApplication(app);
+	});
+
+	describe("Config - advice:true", function () {
+		before(function () {
+			process.env.MM_CONFIG_FILE = "tests/configs/modules/compliments/compliments_advice.js";
+		});
+
+		it("one of less then 35 char options", async function () {
+			// No template available for compliments to pass in here ?
+			await setup(generateAdvice())
+			return app.client.waitUntilTextExists(".compliments .module-content div", oneOf([
```

Comment:
travis doesnt recognize this oneOf... https://travis-ci.com/github/MichMich/MagicMirror/jobs/340236682#L969